gdb/parse.c

```diff
@@ -930,6 +930,7 @@ operator_length_standard (const struct expression *expr, int endpos,
     case BINOP_VAL:
     case UNOP_CAST:
     case UNOP_MEMVAL:
+    case STRUCTOP_ANONYMOUS:
```

Comment:
Unfortunately when adding a new op, it's also best to update `dump_subexp_body_standard` and `print_subexp_standard`.  These are used for debugging if you enable `set debug expression 1`... which normally I never did, but did use while working on the Rust support.

I had forgotten how terrible and weird the gdb expression structure is.  It's hard to imagine why it's this way.